io.github.ladaapp.lada.yaml

```diff
@@ -0,0 +1,48 @@
+id: io.github.ladaapp.lada
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: lada
+finish-args:
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--share=ipc"
+  - "--socket=pulseaudio"
```

Comment:
why does this need sound access?

Reply:
The app includes a simple media player which plays the video including sound